src/sentry_github/plugin.py

```diff
@@ -40,7 +58,7 @@ class GitHubPlugin(IssuePlugin):
     auth_provider = 'github'
 
     def is_configured(self, request, project, **kwargs):
-        return bool(self.get_option('repo', project))
+        return bool(self.get_option('repo', project)) and bool(self.get_option('endpoint', project))
```

Comment:
we'll need this to support legacy projects

that is, we want to say "its configured as long as repo is set", and just default the endpoint value to the same as the initial value (same for github_url).

same as before, if you beat us to it great, otherwise will aim to make changes tomorrow and get this merged in